jetson/nav/gate_search/gateStateMachine.hpp

```diff
@@ -78,7 +78,7 @@ class GateStateMachine
     Odometry centerPoint1;
     Odometry centerPoint2;
 
-    //
+    // Check if we drove through the correct direction
     bool CP1ToCP2CorrectDir;
 
     double gateAdjustmentDist = 0.3;
```

Comment:
This should be a constant.
1. use `const`
2. Should live somewhere more intuitive. I think the config file makes sense. However, another option would be the hpp file but I think the config file makes more sense.

Also, what is a gate adjustment distance?